freqtrade/freqtradebot.py

```diff
@@ -1226,7 +1226,7 @@ def _notify_exit_cancel(self, trade: Trade, order_type: str, reason: str) -> Non
             'profit_ratio': profit_ratio,
             'sell_reason': trade.sell_reason,
             'open_date': trade.open_date,
-            'close_date': trade.close_date,
+            'close_date': trade.close_date or datetime.utcnow(),
```

Comment:
i know `datetime.utcnow()` sounds like the obvious choice - but it will plot the wrong timestamp (it'll create a time object in UTC time - but without timezone, hence making it a "timezone native" object (with the wrong time).

the correct call for this would be `datetime.now(timezone.utc)` ...

Reply:
Hey thank you very much Xmatthias. I copied this from another place (In the Freqtrade codebase)  that did this kind of thing so I was under the impression this was correct behavior. I'll change it when I get home.

Reply:
it'll depend on the usage of that variable.
I know we're using this in certain parts - but afaik, only for counters (where the difference between a prior, identical call is relevant, not the exact time).

Reply:
It should be changed now, but I have no clue how to test it out other than wait until the error (That is now fixed) happens again (i.e in a sell cancel)